uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/BAO/LeavePeriodEntitlementTest.php

```diff
@@ -755,14 +782,36 @@ public function testgetLeavePeriodEntitlementRemainder() {
     );
 
     $params = ['entitlement_id' => $periodEntitlement->id];
-    $result = $periodEntitlement::getLeavePeriodEntitlementRemainder($params);
+    $result = $periodEntitlement::getRemainder($params);
     $this->assertEquals(1, count($result));
     $this->assertEquals(7, $result[0]['remainder']['current']);
+    $this->assertEquals(7, $result[0]['remainder']['current']);
+    $this->assertArrayNotHasKey('future', $result[0]['remainder']);
   }
 
-  public function testgetLeavePeriodEntitlementRemainderWithIncludeFuture() {
-    $absence_period = $this->createAbsencePeriod(date('Y') .'-01-01', date('Y') .'-12-31');
-    $periodEntitlement = $this->createPeriodEntitlement($absence_period->id);
+  public function testGetMultipleLeavePeriodEntitlementRemainder() {
```

Comment:
This name seems a bit weird to me. What about changing it to something like `testGetLeavePeriodEntitlementRemainderWithMultipleRecords`?